src/util.js

```diff
@@ -25,6 +27,48 @@ var Util = {
         message: null
     },
 
+    escapeRegExp: function(string) {
+        return string.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
+    },
+
+    getIndicesOf: function(contentString, searchString) {
```

Comment:
I'll add some comments in here to so that people know that it's ignoring over widget references